lib/DefaultDomainEventNameMapping.js

```diff
@@ -1,14 +1,22 @@
 const { EVENT_TYPE } = require('./eventMessageHeaders');
 
 class DefaultDomainEventNameMapping {
-  constructor() {}
+  constructor(mappings) {
+    this.mappings = mappings;
+    // {
```

Comment:
Remove this code.

Reply:
Fixed with https://github.com/eventuate-tram/eventuate-tram-core-nodejs/pull/13